Gruntfile.js

```diff
@@ -61,8 +61,8 @@ module.exports = function(grunt) {
         entry: './src/HanziWriter.js',
         output: {
           filename: 'dist/hanzi-writer-lib.js',
-          library: 'hanzi-writer',
-          libraryTarget: 'commonjs2'
+          library: 'hanziWriter',
```

Comment:
Could this be called `HanziWriter` instead? That way it would match the casing of what's currently output in the browser build

Reply:
Yep!